controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -478,6 +496,9 @@ public File getConfigurationDir(){
         return configurationDir;
     }
 
+    public File getConfigurationTmpDir() {
```

Comment:
I believe this can be package protected. If so, please make it package protected.

Reply:
Right, made package protected. Thanks for the review.